Item/Item.module.less

```diff
@@ -60,10 +62,12 @@
 
 				&.labelBefore .label {
 					.padding-start-end(0, @agate-item-label-padding);
+					align-self: center;
```

Comment:
Oh.. we have `align` property for `Layout` component.
We could re-write this in Item.js. at line 132,
```
<Layout align={labelPosition === 'before' || labelPosition === 'after' ? "center" : null} orientation={orientation}>
```

Reply:
Good tip